2024-08-07 17:29:28,255 INFO    StreamThr :2461456 [internal.py:wandb_internal():86] W&B internal server running at pid: 2461456, started at: 2024-08-07 17:29:28.254432
2024-08-07 17:29:28,258 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: status
2024-08-07 17:29:28,260 INFO    WriterThread:2461456 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/run-y2h6vath.wandb
2024-08-07 17:29:28,261 DEBUG   SenderThread:2461456 [sender.py:send():382] send: header
2024-08-07 17:29:28,288 DEBUG   SenderThread:2461456 [sender.py:send():382] send: run
2024-08-07 17:29:28,530 INFO    SenderThread:2461456 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files
2024-08-07 17:29:28,530 INFO    SenderThread:2461456 [sender.py:_start_run_threads():1136] run started: y2h6vath with start time 1723022968.25476
2024-08-07 17:29:28,532 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: check_version
2024-08-07 17:29:28,532 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: check_version
2024-08-07 17:29:28,653 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: run_start
2024-08-07 17:29:28,666 DEBUG   HandlerThread:2461456 [system_info.py:__init__():32] System info init
2024-08-07 17:29:28,666 DEBUG   HandlerThread:2461456 [system_info.py:__init__():47] System info init done
2024-08-07 17:29:28,666 INFO    HandlerThread:2461456 [system_monitor.py:start():194] Starting system monitor
2024-08-07 17:29:28,666 INFO    SystemMonitor:2461456 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-07 17:29:28,667 INFO    HandlerThread:2461456 [system_monitor.py:probe():214] Collecting system info
2024-08-07 17:29:28,667 INFO    SystemMonitor:2461456 [interfaces.py:start():190] Started cpu monitoring
2024-08-07 17:29:28,668 INFO    SystemMonitor:2461456 [interfaces.py:start():190] Started disk monitoring
2024-08-07 17:29:28,668 INFO    SystemMonitor:2461456 [interfaces.py:start():190] Started gpu monitoring
2024-08-07 17:29:28,669 INFO    SystemMonitor:2461456 [interfaces.py:start():190] Started memory monitoring
2024-08-07 17:29:28,669 INFO    SystemMonitor:2461456 [interfaces.py:start():190] Started network monitoring
2024-08-07 17:29:28,677 DEBUG   HandlerThread:2461456 [system_info.py:probe():196] Probing system
2024-08-07 17:29:28,679 DEBUG   HandlerThread:2461456 [system_info.py:_probe_git():181] Probing git
2024-08-07 17:29:28,681 DEBUG   HandlerThread:2461456 [system_info.py:_probe_git():189] Probing git done
2024-08-07 17:29:28,681 DEBUG   HandlerThread:2461456 [system_info.py:probe():244] Probing system done
2024-08-07 17:29:28,681 DEBUG   HandlerThread:2461456 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-07T09:29:28.677944', 'startedAt': '2024-08-07T09:29:28.247861', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/Gemma-2B', '--use_lora', 'True', '--init_lora_weights', 'pissa_niter_4', '--target_modules', 'q_proj,v_proj,k_proj,o_proj,gate_proj,down_proj,up_proj', '--lora_rank', '32', '--lora_alpha', '32', '--lora_dropout', '0', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1460.9125000000001, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 2149.869, 'min': 800.0, 'max': 5500.0}, {'current': 2199.673, 'min': 800.0, 'max': 5500.0}, {'current': 2199.213, 'min': 800.0, 'max': 5500.0}, {'current': 2196.554, 'min': 800.0, 'max': 5500.0}, {'current': 2081.563, 'min': 800.0, 'max': 5500.0}, {'current': 1313.065, 'min': 800.0, 'max': 5500.0}, {'current': 2043.974, 'min': 800.0, 'max': 5500.0}, {'current': 1519.092, 'min': 800.0, 'max': 5500.0}, {'current': 2353.557, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 2085.197, 'min': 800.0, 'max': 5600.0}, {'current': 2200.0, 'min': 800.0, 'max': 5600.0}, {'current': 2145.906, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2194.985, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1733.94, 'min': 800.0, 'max': 4300.0}, {'current': 2201.727, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1129.002, 'min': 800.0, 'max': 4300.0}, {'current': 1327.696, 'min': 800.0, 'max': 4300.0}, {'current': 2203.66, 'min': 800.0, 'max': 4300.0}, {'current': 1119.082, 'min': 800.0, 'max': 4300.0}, {'current': 1108.91, 'min': 800.0, 'max': 4300.0}, {'current': 1126.379, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 474.6181411743164}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-07 17:29:28,681 INFO    HandlerThread:2461456 [system_monitor.py:probe():224] Finished collecting system info
2024-08-07 17:29:28,681 INFO    HandlerThread:2461456 [system_monitor.py:probe():227] Publishing system info
2024-08-07 17:29:28,681 DEBUG   HandlerThread:2461456 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-07 17:29:28,682 DEBUG   HandlerThread:2461456 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-07 17:29:28,682 DEBUG   HandlerThread:2461456 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-07 17:29:29,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/conda-environment.yaml
2024-08-07 17:29:29,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/requirements.txt
2024-08-07 17:29:30,450 DEBUG   HandlerThread:2461456 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-07 17:29:30,450 DEBUG   HandlerThread:2461456 [system_info.py:_save_code():90] Saving code
2024-08-07 17:29:30,452 DEBUG   HandlerThread:2461456 [system_info.py:_save_code():111] Saving code done
2024-08-07 17:29:30,452 DEBUG   HandlerThread:2461456 [system_info.py:_save_patches():128] Saving git patches
2024-08-07 17:29:30,474 DEBUG   HandlerThread:2461456 [system_info.py:_save_patches():170] Saving git patches done
2024-08-07 17:29:30,475 INFO    HandlerThread:2461456 [system_monitor.py:probe():229] Finished publishing system info
2024-08-07 17:29:30,476 DEBUG   SenderThread:2461456 [sender.py:send():382] send: files
2024-08-07 17:29:30,476 INFO    SenderThread:2461456 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-07 17:29:30,476 INFO    SenderThread:2461456 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-07 17:29:30,476 INFO    SenderThread:2461456 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-07 17:29:30,478 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: stop_status
2024-08-07 17:29:30,478 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: stop_status
2024-08-07 17:29:30,480 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-07 17:29:30,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/conda-environment.yaml
2024-08-07 17:29:30,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/diff.patch
2024-08-07 17:29:30,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/code/llama/gmm.py
2024-08-07 17:29:30,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/wandb-metadata.json
2024-08-07 17:29:30,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/code
2024-08-07 17:29:30,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/code/llama
2024-08-07 17:29:30,702 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688
2024-08-07 17:29:30,702 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5541', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:30,702 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E968434ADF1404</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:30,735 DEBUG   SenderThread:2461456 [sender.py:send():382] send: telemetry
2024-08-07 17:29:30,735 DEBUG   SenderThread:2461456 [sender.py:send():382] send: config
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: metric
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: telemetry
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: metric
2024-08-07 17:29:30,736 WARNING SenderThread:2461456 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: telemetry
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: telemetry
2024-08-07 17:29:30,736 DEBUG   SenderThread:2461456 [sender.py:send():382] send: config
2024-08-07 17:29:30,737 DEBUG   SenderThread:2461456 [sender.py:send():382] send: exit
2024-08-07 17:29:30,737 INFO    SenderThread:2461456 [sender.py:send_exit():589] handling exit code: 1
2024-08-07 17:29:30,737 INFO    SenderThread:2461456 [sender.py:send_exit():591] handling runtime: 2
2024-08-07 17:29:30,738 INFO    SenderThread:2461456 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 17:29:30,738 INFO    SenderThread:2461456 [sender.py:send_exit():597] send defer
2024-08-07 17:29:30,738 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,738 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-07 17:29:30,739 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,739 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-07 17:29:30,739 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 1
2024-08-07 17:29:30,739 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,739 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-07 17:29:30,739 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,739 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-07 17:29:30,739 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 2
2024-08-07 17:29:30,739 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,739 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-07 17:29:30,739 INFO    HandlerThread:2461456 [system_monitor.py:finish():203] Stopping system monitor
2024-08-07 17:29:30,739 INFO    HandlerThread:2461456 [interfaces.py:finish():202] Joined cpu monitor
2024-08-07 17:29:30,739 DEBUG   SystemMonitor:2461456 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-07 17:29:30,739 INFO    HandlerThread:2461456 [interfaces.py:finish():202] Joined disk monitor
2024-08-07 17:29:30,739 DEBUG   SystemMonitor:2461456 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-07 17:29:30,740 DEBUG   SystemMonitor:2461456 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-07 17:29:30,741 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7
2024-08-07 17:29:30,741 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:30,741 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E968434AF1F685</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:30,796 INFO    HandlerThread:2461456 [interfaces.py:finish():202] Joined gpu monitor
2024-08-07 17:29:30,796 INFO    HandlerThread:2461456 [interfaces.py:finish():202] Joined memory monitor
2024-08-07 17:29:30,796 INFO    HandlerThread:2461456 [interfaces.py:finish():202] Joined network monitor
2024-08-07 17:29:30,797 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,797 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-07 17:29:30,797 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 3
2024-08-07 17:29:30,797 DEBUG   SenderThread:2461456 [sender.py:send():382] send: stats
2024-08-07 17:29:30,797 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,797 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-07 17:29:30,797 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,797 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-07 17:29:30,797 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 4
2024-08-07 17:29:30,797 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,797 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-07 17:29:30,797 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 5
2024-08-07 17:29:30,798 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,798 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-07 17:29:30,798 DEBUG   SenderThread:2461456 [sender.py:send():382] send: summary
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-07 17:29:30,798 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 6
2024-08-07 17:29:30,798 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,798 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-07 17:29:30,798 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,798 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-07 17:29:30,800 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: status_report
2024-08-07 17:29:30,963 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 7
2024-08-07 17:29:30,963 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:30,963 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-07 17:29:30,963 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:30,963 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-07 17:29:30,970 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4
2024-08-07 17:29:30,970 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284247', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:30,970 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E968434F01C3D3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:31,530 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/config.yaml
2024-08-07 17:29:31,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/output.log
2024-08-07 17:29:31,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/wandb-summary.json
2024-08-07 17:29:31,684 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-07 17:29:31,875 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7
2024-08-07 17:29:31,875 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:31,875 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9684390968E87</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:31,875 INFO    wandb-upload_1:2461456 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 17:29:31,912 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688
2024-08-07 17:29:31,912 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5541', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:31,912 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9684392D2031C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:31,912 INFO    wandb-upload_0:2461456 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-07 17:29:32,343 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4
2024-08-07 17:29:32,343 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284247', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:32,343 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96843A3D8EEF0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:32,745 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 8
2024-08-07 17:29:32,745 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: poll_exit
2024-08-07 17:29:32,745 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:32,745 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-07 17:29:32,746 DEBUG   SenderThread:2461456 [sender.py:send_request():409] send_request: defer
2024-08-07 17:29:32,746 INFO    SenderThread:2461456 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-07 17:29:32,746 INFO    SenderThread:2461456 [job_builder.py:build():298] Attempting to build job artifact
2024-08-07 17:29:32,747 INFO    SenderThread:2461456 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-07 17:29:32,764 INFO    SenderThread:2461456 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-07 17:29:32,769 INFO    SenderThread:2461456 [sender.py:transition_state():617] send defer: 9
2024-08-07 17:29:32,769 DEBUG   HandlerThread:2461456 [handler.py:handle_request():146] handle_request: defer
2024-08-07 17:29:32,769 INFO    HandlerThread:2461456 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-07 17:29:32,769 DEBUG   SenderThread:2461456 [sender.py:send():382] send: artifact
2024-08-07 17:29:33,315 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=068ea1addbdf9e319270ea002412210a4dbcac4769ae35a9a2a3589f5998c380: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=068ea1addbdf9e319270ea002412210a4dbcac4769ae35a9a2a3589f5998c380
2024-08-07 17:29:33,315 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:33,315 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E96843E6918C5A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:33,370 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=a3c7a5881719c64db5e763fa34462eaddaf2c1070071994e72d3248f312b0e23: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=a3c7a5881719c64db5e763fa34462eaddaf2c1070071994e72d3248f312b0e23
2024-08-07 17:29:33,370 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'kxXsJy8yCxj0IM1+NfsfyA==', 'Content-MD5': 'kxXsJy8yCxj0IM1+NfsfyA==', 'Content-Type': 'application/json', 'Content-Length': '31085', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:33,370 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8</Resource><Region>wandb-local</Region><RequestId>17E96843E68122BF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:33,531 INFO    Thread-12 :2461456 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240807_172928-y2h6vath/files/output.log
2024-08-07 17:29:33,983 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f1ee8781bae9e3ba2d7393e6c1245628001fd187eef147bb48a604192f14e6c7
2024-08-07 17:29:33,983 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:33,983 ERROR   wandb-upload_1:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E968440E68508D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:34,150 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=538714460de886ff64cb7f0fd4a4fae9984692a110e67cc6e5c1102f920f4688
2024-08-07 17:29:34,150 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5541', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:34,150 ERROR   wandb-upload_0:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E96844186C5F48</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:34,422 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=068ea1addbdf9e319270ea002412210a4dbcac4769ae35a9a2a3589f5998c380: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=068ea1addbdf9e319270ea002412210a4dbcac4769ae35a9a2a3589f5998c380
2024-08-07 17:29:34,422 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:34,422 ERROR   wandb-upload_3:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/501/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9684428749610</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:34,611 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=a3c7a5881719c64db5e763fa34462eaddaf2c1070071994e72d3248f312b0e23: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092933Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=a3c7a5881719c64db5e763fa34462eaddaf2c1070071994e72d3248f312b0e23
2024-08-07 17:29:34,611 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'kxXsJy8yCxj0IM1+NfsfyA==', 'Content-MD5': 'kxXsJy8yCxj0IM1+NfsfyA==', 'Content-Type': 'application/json', 'Content-Length': '31085', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:34,611 ERROR   wandb-upload_4:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/41/501/9315ec272f320b18f420cd7e35fb1fc8</Resource><Region>wandb-local</Region><RequestId>17E968443240D41E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-07 17:29:34,633 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/y2h6vath/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240807%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240807T092930Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=7c9f7016588366ba32ceb26c7f2054c3919fd5e2faab87eac18024c5aae072f4
2024-08-07 17:29:34,633 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '284247', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-07 17:29:34,633 ERROR   wandb-upload_2:2461456 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/y2h6vath/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/y2h6vath/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E96844332B59A4</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
